package io.artoo.lance.thread;

public final class TaskException extends RuntimeException {
  public TaskException(final Throwable cause) {
    super(cause);
  }
}
